vim/ftplugin/gitcommit.vim

```diff
@@ -3,3 +3,4 @@ setlocal spell
 setlocal colorcolumn=51
 
 nnoremap <buffer> gE :call ExpandShortGitShaIntoFullGitSha()<CR>
+silent CocDisable
```

Comment:
I tried setting `"coc.source.buffer.disableSyntaxes": ["gitcommit"]` but it didn't do anything, so I resorted to this. It's fine.

Reply:
The setting I have is `"workspace.ignoredFiletypes": ["gitcommit"],` which seems to work 🙂